script.js

```diff
@@ -0,0 +1,31 @@
+const ham = document.querySelector('.hamburger');
+const closeMenu = document.querySelector('.close');
+const sections = document.querySelectorAll('section');
+const menuContainer = document.querySelector('.menu-container');
+const mobileLinKItem = document.querySelectorAll('.mobile-link-item');
+
+ham.addEventListener('click', () => {
+  for (let i = 0; i < sections.length; i += 1) {
+    sections[i].classList.remove('blur');
+  }
+
+  menuContainer.style.display = 'block';
```

Comment:
- [ ] [OPTIONAL] great job setting up the mobile menu :clap: , I have just one suggestion, here you see we are repeating the same block of code multiple times 3 to be exact, this is the best place to use a function to group the repeating code together and follow the DRY principle, how we can do that is :point_down: 
```
function toggleMobileMenu(value){
  for (let i = 0; i < sections.length; i += 1) {
    sections[i].classList.remove('blur');
  }

  menuContainer.style.display = `${value}`;
}
```
then we can use this block of code however many times without repeating a lot of code like this :point_down: 
```
ham.addEventListener('click', ()=>{
  toggleMobileMenu("block")
});

closeMenu.addEventListener('click', ()=>{
  toggleMobileMenu("none")
});

for (let i = 0; i < mobileLinKItem.length; i += 1) {
  mobileLinKItem[i].addEventListener('click', ()=>{
    toggleMobileMenu("none")});
}
```